buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -342,6 +342,10 @@ private void configure() {
         if (Version.fromString(version).getMajor() >= 6) {
             config.put("cluster.routing.allocation.disk.watermark.flood_stage", "1b");
         }
+        if (Version.fromString(version).onOrAfter(Version.fromString("6.5.0"))) {
+            config.put("discovery.type", "zen2");
```

Comment:
zen2 is the default. Why is this change necessary? These tests are probably already running with Zen2 unless explicitly disabled in some place

Reply:
Also `zen2` is only available in ≥7.0 so `onOrAfter(6.5.0)` looks wrong.

Reply:
My goal was to avoid the 3s delay. Will look into it, thanks !

Reply:
I removed `discovery.type`, sorry was looking at an older PR.